eng/common/scripts/job-matrix/job-matrix-functions.ps1

```diff
@@ -190,6 +190,12 @@ function ParseFilter([string]$filter) {
     }
 }
 
+function GetMatrixConfigFromYaml([String] $yamlConfig)
+{
+    $config = $yamlConfig | ConvertFrom-Yaml
+    return GetMatrixConfigFromJson (ConvertTo-Json $config)
```

Comment:
Nit: this might be a little cleaner?

```suggestion
    $config = $yamlConfig | ConvertFrom-Yaml | ConvertTo-Json
    return GetMatrixConfigFromJson $config
```

Reply:
Thinking about this more: Can you refactor the `GetMatrixConfigFromJson` function to be a simple json conversion that calls into an inner `GetMatrixConfig` function? That way you can just call that function directory from the `GetMatrixConfigFrom<config format>` functions.

Reply:
Or even simpler, just have a single function `GetMatrixConfig` and always run the string through `ConvertFrom-Yaml`. Yaml is a superset of json so `ConvertFrom-Yaml` will be able to parse either a yaml config file or a json config file.